JenkinsfileTrigger

```diff
@@ -64,84 +67,87 @@ node('master') {
                                 echo "topic=${MAIN_TOPIC}.ci.pipeline.package.ignore" >> ${WORKSPACE}/job.properties
                             else
-                                echo "topic=${MAIN_TOPIC}.ci.pipeline.package.queued" >> ${WORKSPACE}/job.properties
-                                touch ${WORKSPACE}/trigger.downstream
+                                # Verify this is a package we are interested in
+                                valid=0
+                                for package in $(cat ${PROJECT_REPO}/config/package_list); do
+                                    if [ "${package}" = "${fed_repo}" ]; then
+                                        valid=1
+                                        break
+                                    fi
+                                done
+
+                                if [ $valid -eq 0 ]; then
+                                    echo "Not a package we are interested in"
+                                    echo "topic=${MAIN_TOPIC}.ci.pipeline.package.ignore" >> ${WORKSPACE}/job.properties
+                                else
+                                    echo "topic=${MAIN_TOPIC}.ci.pipeline.package.queued" >> ${WORKSPACE}/job.properties
+                                    touch ${WORKSPACE}/trigger.downstream
+                                fi
                             fi
-                        fi
-                        exit
-                    '''
-                    def job_props = "${env.WORKSPACE}/job.properties"
-                    def job_props_groovy = "${env.WORKSPACE}/job.properties.groovy"
-                    convertProps(job_props, job_props_groovy)
-                    load(job_props_groovy)
-
-                    // Populate message Properties
-                    messageProperties = "topic=${topic}\n" +
-                                        "build_url=${BUILD_URL}\n" +
-                                        "build_id=${BUILD_ID}\n" +
-                                        "branch=${branch}\n" +
-                                        "ref=fedora/${branch}/x86_64/atomic-host\n" +
-                                        "rev=${fed_rev}\n" +
-                                        "repo=${fed_repo}\n" +
-                                        "namespace=${fed_namespace}\n" +
-                                        "username=fedora-atomic\n" +
-                                        "test_guidance=''\n" +
-                                        "status=${currentBuild.currentResult}"
-                    messageContent = ''
-
-                    // Send message org.centos.prod.ci.pipeline.package.queued or .ignore on fedmsg
-                    sendMessage(messageProperties, messageContent)
+                            exit
+                        '''
+                    }
+
+                    def jobproperties_map = readProperties file: "{env.WORKSPACE}/job.properties"
```

Comment:
I don't think we need this for the message properties we send we need it for shell files only that is the time when we do the conversion